.markdownlint.yaml

```diff
@@ -1,9 +1,22 @@
-MD001: false
-
+# MD010/no-hard-tabs : Hard tabs : https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md010.md
+#
+# The rendered markdown is viewed in github and
+# the Docusaurus site which render hard-tabs fine.
+# It is annoying to have to deal with so many spaces
+# specifically in golang code blocks.
 MD010: false
 
+# MD013/line-length : Line length : https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md013.md
 MD013:
+  # Applying line length to code blocks
+  # makes the code look worse sometimes.
   code_blocks: false
+  # Applying line length to tables makes
+  # it more difficult to edit.
   tables: false
```

Comment:
Is it really necessary? Why would you want that?

Reply:
not following. Both these lines are disabling line length checks on `code_blocks` and `tables`

I guess this show I should change my rhetoric too `Disabling line length check on code blocks. Applying against code blocks makes the code look work sometimes`?

Reply:
Sorry my comment was unclear.
I would disable line length checker. I don't care if my markdown line is 123 character long. I don't see how it would improve lisibility.

Reply:
It doesn't have to do with the rendering of the markdown, more so the developer experience. I use a basic text editor. Viewing long lines is a pain. If you're against I can disable, but it IMO it makes for a much better experience when editing markdown.

Reply:
If it helps you, that's OK for me :)